tests/kernel/wave/attention/extend_attention_test.py

```diff
@@ -258,6 +259,30 @@ def create_inputs(
     qo_indptr[1 : B + 1] = torch.cumsum(b_seq_len_extend[:B], dim=0)
     logit_cap = 30.0
 
+    b_seq_mask_len = b_seq_len_extend * b_seq_len
+    custom_mask = device_full(
+        (b_seq_mask_len.sum().item(),), fill_value=1, dtype=torch.int8
+    )
+    mask_offsets = device_zeros((B + 1,), dtype=torch.int32)
+    mask_offsets[1 : B + 1] = torch.cumsum(b_seq_mask_len[:B], dim=0)
+    for i in range(B):
+        causal_mask = (
```

Comment:
I think it's more valuable if we have custom mask as randoms

Reply:
Pursuant to our offline discussion, I'm adding a warning for untested random mask